static/js/unread.js

```diff
@@ -264,9 +264,55 @@ exports.num_unread_for_person = function (user_ids_string) {
     return unread_privates.get(user_ids_string).num_items();
 };
 
+exports.current_stream_topic = function () {
+    var selected_stream;
+    var selected_topic;
+    if (narrow.filter() === undefined || narrow.filter()._operators["1"] === undefined) {
+         var message_header = $(".selected_message").closest(".recipient_row").find(".message_header");
```

Comment:
Are you sure there's no easy way to find out the current stream and topic?  What does `stream_list.js` use?